common/lang_detection.go

```diff
@@ -26,6 +31,16 @@ const (
 	IgnoredProgrammingLanguage ProgrammingLanguage = "ignored"
 )
 
+func MapOdigosToSemConv(odigosPrograminglang ProgrammingLanguage) string {
+	fmt.Println(semconv.TelemetrySDKLanguageNodejs.Value.AsString())
+	switch odigosPrograminglang {
+	case JavascriptProgrammingLanguage:
+		return semconv.TelemetrySDKLanguageNodejs.Value.AsString()
+	default:
```

Comment:
While we currently only care about python and nodejs, this function might be called in the future with other odigos programming languages which has no otel equivalents (`mysql`, `nginx`, etc).

Should we handle this here and make sure we only take what we can map correctly?

Reply:
added a comment for this as discussed